src/NBomber/Domain/Assertion.fs

```diff
@@ -4,8 +4,14 @@ module internal NBomber.Domain.Assertion
 open NBomber.Contracts
 open NBomber.Errors
 
-let create (assertions: IAssertion[]) = 
-    assertions |> Array.map(fun x -> x :?> Assertion)
+let create : IAssertion[] -> Assertion[] = 

```

Comment:
for such name alias cases, usually, you will see that ppl omit type annotations
let create = Array.map ....